apps/api/src/controllers/applicationController.ts

```diff
@@ -88,7 +102,7 @@ export const editApplication = async ({
 
 	if (!formattedResult.success) return formattedResult;
 
-	return await applicationRepo.editApplication({ id, update: formattedResult.data });
+	return await applicationRepo.editApplication({ id, update: formattedResult.data, shouldKeepState: shouldKeepState });
```

Comment:
We can just pass `shouldKeepState`

```suggestion
	return await applicationRepo.editApplication({ id, update: formattedResult.data, shouldKeepState });
```